cmd/spire-server/cli/agent/count.go

```diff
@@ -49,6 +123,12 @@ func (c *countCommand) Run(ctx context.Context, _ *commoncli.Env, serverClient u
 }
 
 func (c *countCommand) AppendFlags(fs *flag.FlagSet) {
+	fs.Var(&c.selectors, "selector", "A colon-delimited type:value selector. Can be used more than once")
+	fs.StringVar(&c.attestationType, "attestationType", "", "The SPIFFE ID of the nodes to count")
+	fs.Var(&c.canReattest, "canReattest", "Filter based on string received, 'true': agents that can reattest, 'false': agents that can't reattest, other value will return all.")
+	fs.Var(&c.banned, "banned", "Filter based on string received, 'true': banned agents, 'false': not banned agents, other value will return all.")
+	fs.StringVar(&c.expiresBefore, "expiresBefore", "", "A date that indicates the time it should expired before, (format: YYYY-MM-DD)")
```

Comment:
In the same way that we make it explicit in the description of other flags that the flag will apply a filter, I think that we should make it explicit here. Something like:
`Filter by expiration time (format: "2006-01-02 15:04:05 -0700 -07").`
I've commented about the format separately.

The same applies to other flags added that don't mention that a filter will be applied.